src/components/docs-anchor/index.tsx

```diff
@@ -1,60 +0,0 @@
-import Link from 'next/link'
-import classNames from 'classnames'
-import { useCurrentProduct } from 'contexts'
-import InlineLink from 'components/inline-link'
-import s from 'components/inline-link/inline-link.module.css'
-
-const DocsAnchor = ({
-	href,
-	children,
-	...rest
-}: JSX.IntrinsicElements['a']) => {
-	const currentProduct = useCurrentProduct()
-	const passableProps = {
-		...rest,
-		// TODO: we probably want to make use of the InlineLink component in the future instead of just adopting its styles here. Tackle that when we have bandwidth to VQA lots of links.
-		className: classNames(rest.className, s.root),
-	}
-
-	if (!href) {
-		return <a {...passableProps}>{children}</a>
-	}
```

Comment:
This case mainly applies to "target" elements generated by our `anchor-links` remark plugin. I've modified the `InlineLink` component to fallback to using `#` as the `href` if `href` is falsy. This is a temporary measure, we should likely not generate `<a />` tags without `href`. Asana task to address this in the context of `anchor-links`: 🎟️ [Write RFC for using ids in anchor-links plugin rather than injecting HTML](https://app.asana.com/0/1100423001970639/1202488740132127/f).